model/src/main/java/com/epam/esm/dao/querybuilder/GiftCertificateQueryBuilder.java

```diff
@@ -0,0 +1,55 @@
+package com.epam.esm.dao.querybuilder;
+
+import com.epam.esm.entity.filter.SearchFilter;
+import org.apache.commons.lang.StringUtils;
+import org.springframework.stereotype.Component;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@Component
+public class GiftCertificateQueryBuilder implements QueryBuilder {
+
+    public String buildFilteredSelectQuery(String selectQuery, SearchFilter searchFilter) {
```

Comment:
Please, always use @Override when you override some method. It generally improves the readability of your code